Add README.md

jonathantneal opened the pull request.

zenorocha commented on README.md:
```diff
@@ -0,0 +1,55 @@
+# Marble
```
WeDeploy.com

zenorocha commented on README.md:
````diff
@@ -0,0 +1,55 @@
+1. From the [node/electric] directory, install the local dependencies:
+
+   ```sh
+   npm rium 
````
npm run

zenorocha commented:
Can you remove this readme as well? https://github.com/wedeploy/wedeploy.com/tree/master/node/electric

jonathantneal commented:
Thanks, @zenorocha, @pragmaticivan! I’ve fixed the typo, added a badge for CI, added a badge for Slack, replaced the blockquote with more a meaningful description of the repository and WeDeploy, and then removed the README.md file within electric. I’ve also tried to generally make it look very attractive.

Would you please re-review?

zenorocha commented on README.md:
```diff
@@ -0,0 +1,75 @@
+[cli-url]: https://ci.wedeploy.com/job/wedeploy/job/wedeploy.com/job/master/
+[cli-img]: https://ci.wedeploy.com/buildStatus/icon?job=wedeploy/wedeploy.com/master
+[irc-url]: https://chat.wedeploy.com/
+[irc-img]: https://img.shields.io/badge/chat-slack-blue.svg
```
Use this one instead: https://chat.wedeploy.com/badge.svg

jonathantneal replied:
Done.
